src/neptune_lightgbm/impl/__init__.py

```diff
@@ -236,9 +240,9 @@ def create_booster_summary(
         list_trees (list): Defaults to None. Indices of the target tree to visualize.
             Works only if ``log_trees`` is set to ``True``.
             See `lightgbm.plot_tree`_ for details.
-        log_trees_as_dataframe (bool): Defaults to True.
+        log_trees_as_dataframe (bool): Defaults to False.
             Parse the model and log trees in the easy-to-read pandas DataFrame format.
-            Works only for ``lgb.Booster``.
+            Works only for ``lgb.Booster``, and resultant dataframe should be smaller than 32MB to be logged.
```

Comment:
@Raalsky - does [@neptune-ai/neptune-client#1070 ](https://github.com/neptune-ai/neptune-client/releases/tag/0.16.12) make this obsolete?